docs/package.json

```diff
@@ -32,6 +32,7 @@
     "react-hot-loader": "3.0.0-beta.6",
     "recast": "^0.11.17",
     "style-loader": "^0.16.1",
+    "webpack": "^2.6.1",
```

Comment:
Right.  A little confusing, but there's two levels to the repo.  Webpack gets in installed in the outer npm install.  I'll take care of that.